packages/api-client/src/auth/AuthAPI.ts

```diff
@@ -77,8 +77,10 @@ export class AuthAPI {
   }
 
   public async postLogin(loginData: LoginData): Promise<AccessTokenData> {
+    const {verificationCode, ...rest} = loginData;
     const login = {
-      ...loginData,
+      ...rest,
+      ...(verificationCode && {verification_code: verificationCode}),
```

Comment:
isn't it better if we do it like this?
what is added to the `login` object if `&&` condition is false? 
```
if (verificationCode) {
  login.verification_code = verificationCode;
}
```

Reply:
these are the same 
https://blog.agney.dev/optional-spreading/